modules/logging-loki-log-access.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-loki.adoc
+
+:_content-type: CONCEPT
+[id="logging-loki-log-access_{context}"]
+= Configuring Loki log access
+
+In the {logging} 5.8 and later versions, the Loki Operator supports managing access to logs through cluster wide policies, or by namespace. Users who are members of any group specified in the `adminGroups` field are considered admins in the authorization process. Admin access to logs is not restricted by namespace.
+
+.Example admin group settings
+[source,yaml]
+----
+apiVersion: loki.grafana.com/v1
+kind: LokiStack
+metadata:
+  name: logging-loki
+  namespace: openshift-logging
+spec:
+  tenants:
+    mode: openshift-logging # <1>
+    openshift:
+      adminGroups: # <2>
+      - cluster-admin
+      - custom-admin-group
+----
+<1> Custom admin groups are only available in this mode.
+<2> Overrides the default groups (`system:cluster-admin`, `cluster-admin`, `dedicated-admin`)
+
+
+
+.Example cluster wide log access setting
+[source,yaml]
+----
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: logging-all-application-logs-reader
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-logging-application-view # <1>
+subjects: # <2>
+- kind: Group
+  name: system:authenticated # <3>
+  apiGroup: rbac.authorization.k8s.io
+----
+<1> Additional `ClusterRoles` are `cluster-logging-infrastructure-view`, and `cluster-logging-audit-view`.
```

Comment:
We could also have a RoleBinding to 'cluster-logging-application-view' ClusterRole which is namespace restricted. This can be used for providing logs access to regular users who should not have Cluster-wide access.